### PR TITLE
Medical overhaul

### PR DESCRIPTION
**Pull request by grinkgo**
## About The Pull Request

This pr looks to alleviate a lot of the problems I have with the current state of medical and the meta, as well as make some quality of life adjustments that I think are necessary. For the most part, I want to eliminate the medic surgery meta because I think it's honestly retarded. In return, I'll attempt to give medics some more options in their vendor, and I'm open to suggestions on what else should be added. 

## Why It's Good For The Game

Why does anyone play MO if you can just be a medic, buy a SURGERY VEST from your vendor, rob the OR and do field surgery yourself. Plus with all the OP chems medics get at roundstart, what even is the point of sending marines back to medbay.

## Changelog
:cl:
balance: Corpsmen can now buy a wider variety of pills and injectors, including bicamera and keloderm.
add: Fully stocked medkit pouch added to Corpsmen closet.
tweak: Medkit pouches now have advanced kits.
del: Corpsmen equipment vendor no longer vends the surgical vest.
tweak: Surgical vests now spawn with a surgery tray's worth of tools.
tweak: MarineMeds are now accessible to all marine personnel. 
tweak: Changed around some items in the MarineMed and in the corpsman vendor.
tweak: Advanced combat autoinjectors now have 5u of tramadol instead of 5u of oxycodone.
tweak: Oxycodone, Meralyne, and Dermaline OD changed from 19.8u to 15u.
/:cl:

### CODE REVIEW

**Comment by usnpeepoo** [comment]
>keloderm and not derkel
atomize this pr. Now.

**Comment by InterroLouis** [comment]
Why not use hydrocodone for the advanced combat injector instead? It's the same painkiller level as the oxycodone, but doesn't bonk your stamina.

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -316,6 +316,13 @@
 		/datum/reagent/consumable/sugar = 30,
 	)
 
+/obj/item/reagent_containers/hypospray/advanced/tricordrazine
+	name = "Tricordrazine hypospray"
+	desc = "A hypospray loaded with tricordrazine"
```
Comment (XSlayer300) [inline code/game/objects/items/reagent_containers/hypospray.dm:321]:
```suggestion
	desc = "A hypospray loaded with tricordrazine."
```

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -241,6 +241,16 @@
 	new /obj/item/reagent_containers/glass/bottle/ironsugar(src)
 	new /obj/item/reagent_containers/glass/bottle/ironsugar(src)
 
+/obj/item/storage/syringe_case/tricordrazine
+	name = "syringe case (tricordrazine)"
+	desc = "It's a medical case for storing syringes and bottles. This one contains Tricordrazine"
```
Comment (XSlayer300) [inline code/game/objects/items/storage/firstaid.dm:246]:
```suggestion
	desc = "It's a medical case for storing syringes and bottles. This one contains Tricordrazine."
```

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-6). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**